bash_test.sh

```diff
@@ -208,6 +208,21 @@ EOF
   return $ret
 }
 
+# Test that we don't use %n in C++ code to avoid using it in printf and scanf.
+# This test is not very precise but in cases where "module n" is needed we would
```

Comment:
Perhaps write why %n should be avoided (security? platform compatibility?) for future reference

Reply:
Added a mention that it is not allowed in Android anymore.